model/src/main/proto/javaclasses/mealorder/c/events.proto

```diff
@@ -227,10 +227,10 @@ message PurchaseOrderSent {
     // Sending time.
     google.protobuf.Timestamp when_sent = 2;
 
-    // An E-mail address of the sender.
+    // An E-mail address of a sender.
     spine.net.EmailAddress sender_email = 3;
 
-    // An E-mail address of the recipient.
+    // An E-mail address of a recipient.
     spine.net.EmailAddress vendor_email = 4;
 }
 
```

Comment:
Maybe we should add more detailed explication? 
Purchase order validation may complete with failure if some orders contain large amount of the same dish. That may indicate user mistake.  Such orders are attached to this event.